libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -80,33 +98,73 @@ class NavigationView : ConstraintLayout {
     private val mapView: MapView by lazy {
         MapView(context, mapInitOptions)
     }
-    private val lifeCycleOwner: LifecycleOwner
     private val binding = MapboxLayoutDropInViewBinding.inflate(
         LayoutInflater.from(context),
         this
     )
     private lateinit var navigationViewOptions: NavigationViewOptions
 
-    private lateinit var accessToken: String
-    private lateinit var maneuverView: View
-    private lateinit var speedLimitView: View
-    private lateinit var soundButtonView: View
-    private lateinit var tripProgressView: View
-    private lateinit var recenterButtonView: View
-    private lateinit var routeOverviewButtonView: View
-    private val mapInitOptions: MapInitOptions
-    private val navigationLocationProvider = NavigationLocationProvider()
+    private val onPositionChangedListener = OnIndicatorPositionChangedListener { point ->
+        mapView.getMapboxMap().getStyle()?.let { style ->
+            routeLineViewModel.positionChanged(point, style)
+        }
```

Comment:
Why do we not have a `MapboxRouteLineComponent`? Not a blocker, just wondering if this is intentional or planned as follow up.

Reply:
The route line just got merged yesterday, so probably was missed.

Reply:
Okay, feel free to add here or cut a ticket :+1:

Reply:
Added `RouteLineUIComponent` and `RouteArrowUIComponent`
@LukasPaczos